src/main/kotlin/no/nav/tilleggsstonader/sak/ekstern/journalføring/AutomatiskJournalføringController.kt

```diff
@@ -0,0 +1,61 @@
+package no.nav.tilleggsstonader.sak.ekstern.journalføring
+
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import no.nav.tilleggsstonader.kontrakter.felles.Stønadstype
+import no.nav.tilleggsstonader.sak.fagsak.domain.PersonIdent
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.Feil
+import no.nav.tilleggsstonader.sak.infrastruktur.sikkerhet.SikkerhetContext
+import no.nav.tilleggsstonader.sak.util.FnrUtil.validerIdent
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping(
+    path = ["/api/ekstern/automatisk-journalforing"],
+    consumes = [MediaType.APPLICATION_JSON_VALUE],
+    produces = [MediaType.APPLICATION_JSON_VALUE],
+)
+class AutomatiskJournalføringController(private val automatiskJournalføringService: AutomatiskJournalføringService) {
+
+    /**
+     * Skal bare brukes av tilleggsstonader-soknad-api for å vurdere om en journalføring skal automatisk ferdigstilles
+     * eller manuelt gjennomgås.
+     */
+
+    @PostMapping("kan-opprette-behandling")
```

Comment:
Vi snakket en gang om at vi kanskje bare kunde ha en endepunkt som var "håndter søknad"
Den endepunkten kan selv ansare for om den kan opprette eller ikke, hvis:
- Ja -> ferdigstill journalpost, opprett behandling og oppgave
- Nei -> opprett oppgave for journalpost? 

Sånn at søknad egentlige bare varsler om at den har opprettet en journalpost for søknad X, så får sak behandle resten 👀 
Hva tror du?

Reply:
Enig at man burde ha det sånn, men synes ikke det er verdt å bruke tid på omskrivingen nå 🤷 Nå kan jeg bare kopiere tester og kode, uten å måtte tenke noe særlig.